test/e2e/framework/helper/validation/certificates/certificates.go

```diff
@@ -138,6 +138,22 @@ func ExpectCertificateOrganizationToMatch(certificate *cmapi.Certificate, secret
 	if certificate.Spec.Subject != nil {
 		expectedOrganization = certificate.Spec.Subject.Organizations
 	}
+	if certificate.Spec.LiteralSubject != "" {
```

Comment:
This logic was missing, the expected organisation can also originate from the LiteralSubject.